root/etc/e-smith/db/configuration/migrate/Remove_logs_delegation

```diff
@@ -0,0 +1,17 @@
+{
+    #
+    # Remove the `logs` delegation to users
+    #
+    use esmith::ConfigDB;
+
+    my $db = esmith::ConfigDB->open();
```

Comment:
Do not reopen ConfigDB. Use `$DB` instead

```suggestion
```

Ref http://docs.nethserver.org/projects/nethserver-devel/en/v7/databases.html#migrate-fragments

Reply:
done